docs/using_tilt_extensions.md

```diff
@@ -15,9 +15,17 @@ Once you've found an extension, using it couldn't be easier! Say we wanted to us
 load('ext://hello_world', 'hi')
 ```
 
-`ext://hello_world` is resolved to https://github.com/windmilleng/tilt-extensions/blob/master/hello_world/Tiltfile by Tilt. When the extension is loaded in to your project for the first time Tilt writes the Tiltfile in to the `tilt_modules` directory in your project. So in this case Tilt would write the contents of `ext://hello_world` to `tilt_modules/hello_world`.
+`ext://hello_world` is resolved to https://github.com/windmilleng/tilt-extensions/blob/master/hello_world/Tiltfile by Tilt. When the extension is loaded into your project for the first time, Tilt copies the remote Tiltfile into the `tilt_modules` directory of local your project. So in this case Tilt would write the contents of `ext://hello_world` to `tilt_modules/hello_world`.
+
+`hi` is the name of the function defined in [hello_world/Tiltfile](https://github.com/windmilleng/tilt-extensions/blob/master/hello_world/Tiltfile) and now made available to your local project Tiltfile. Simply call `hi()` anywhere to execute it, printing "Hello world!".
 
 The `tilt_modules` directory should be committed to your repo (**not** `gitignore`'d) to ensure that your teammates don't have to download it in the future, and to ensure that everyone on your team is using the same version of the extension.
```

Comment:
I think something along these lines would be good: "Once Tilt has downloaded an extension, it will not update it, so you won't get surprise breakages. We will soon be implementing support for intentionally updating its local copies of extensions."

(or adjust the "soon" depending on where it might be triaged, but it should be on the list, even if just to define a manual process)

Reply:
or maybe that's out of scope of this PR? In which case, FFTI and I can post a separate PR.  Mostly bringing it up because it was the main thing I was hoping for out of the "we should document lack of versioning" conversation yesterday.

Reply:
What is "FFTI"?

Reply:
Feel Free To Ignore

Reply:
I added your first sentence, but not the second sentence. Let's add the second sentence, but not use "soon", and instead link to a GitHub issue where we can use that as the source of truth of expressing "soon-ness". 

I figure we can add the second sentence once we actually get the feature out the door and it is more public. So maybe it would be a bit weird to have an open GitHub issue that references a non-advertised feature right now? (I don't think it _really_ matters, since this very discussion is in a public repo. But just to keep the https://github.com/windmilleng/tilt issue tracker somewhat clean I think would make sense.)